### DISCUSSION TITLE
smart-pointers/deref

### DISCUSSION THREAD

**Discussion by giscus** [sunface/rust-by-practice#248] [Book Comments]
# smart-pointers/deref

Learning Rust By Practice, narrowing the gap between beginner and skilled-dev with challenging examples, exercises and projects.

https://zh.practice.rs/smart-pointers/deref.html

**Comment by 9a-aaaaaaaa** [comment]
i am a looser!

**Reply by 17998384** [reply]
yeah，you are real loser
